inst/report/script.js

```diff
@@ -0,0 +1,23 @@
+function openTab(event, name) {
+    var tabcontent = document.getElementsByClassName("tabcontent");
+    for (var i = 0; i < tabcontent.length; i++) {
+        tabcontent[i].style.display = "none";
+    }
+
+    var tablinks = document.getElementsByClassName("tablinks");
+    for (var i = 0; i < tablinks.length; i++) {
+        tablinks[i].className = tablinks[i].className.replace(" active", "");
+    }
+
+    // tabcontent.forEach((el) => {
```

Comment:
These lines commented out too?

Reply:
Thanks - was an alternative implementation attempt though it did no work and I'm not sure it's any better. Removed in 5eeda01